UI/web-src/ui/components/systemSettingsPage/systemSettingsPage.js

```diff
@@ -116,6 +116,13 @@ class SystemSettingsPageController {
     valueChanged(systemSetting) {
         this.changedValues[systemSetting.key] = systemSetting.value;
     }
+
+    setSection(section) {
+        this.$state.go(section.name, {
+            location: 'replace',
+            notify: false
```

Comment:
I don't think you need to pass these options through, this will prevent the back button from working.